runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -93,6 +93,8 @@
 	private boolean notifyPosition = false;
 	private boolean notifyHitpoints = true;
 	private boolean notifyPrayer = true;
+	private boolean notifyLowEnergy = true;
+	private boolean notifyHighEnergy = true;
```

Comment:
These names seem very strange, since they're backwards from what they actually do. They're basically only ever checked as `if (!notify) notifier.notify();`

Instead, flip the values and checks for these booleans so they start as `false` and trigger notifications when they're `true`.

Reply:
I was following the way prayers and hitpoints was implemented, but I switched them around and made the var names a bit more descriptive.